### ISSUE TITLE
Support modify font

### ISSUE THREAD

**Issue by Kinmfer** [kingToolbox/WindTerm#148]
Could you add a function to support modify font?Thank you very much

**Comment by kingToolbox** [comment]
Currently you can change the font by modifying the `text.default` in the `scheme.theme` file. Please read [this comment](https://github.com/kingToolbox/WindTerm/issues/59#issuecomment-797263696) for details. The setting interface will be implemented in subsequent versions. Thank you.

**Comment by Kinmfer** [comment]
> Currently you can change the font by modifying the `text.default` in the `scheme.theme` file. Please read [this comment](https://github.com/kingToolbox/WindTerm/issues/59#issuecomment-797263696) for details. The setting interface will be implemented in subsequent versions. Thank you.

okok ,thank you very much,I like windTerm!

**Comment by almas1992** [comment]
Does it support font ligatures features？

**Comment by kingToolbox** [comment]
Yes, `font ligatures` is supported. But this feature is disabled internally.

In order to support drawing very long lines, such as 1GB single line, WindTerm/WindEdit uses an efficient algorithm to calculate the text width of each line, and `font ligatures` will bring more complicated rules to the calculation, such as judging whether each character can be combined with the next character. This will greatly reduce calculating performance, so this feature is disabled. 

If there is a need, WindTerm/WindEdit can provide an option to enable the `font ligatures` feature when needed, after all, most of the time there is no need to deal with very long lines.

**Comment by kingToolbox** [comment]
@Kinmfer 
My pleasure. :smile:

**Comment by JavaZeroo** [comment]
Any Update?

**Comment by kingToolbox** [comment]
Support for ligatures slows down character width calculations, which is why it hasn't been implemented. I'll reconsider this feature in version `3.x`, but cannot guarantee its inclusion.

Personally, I don't see much value in it, but I'm just a developer. Whether a feature is useful is up to the users. I remain neutral. Generally, I try to provide features users request.

**Comment by sTiKyt** [comment]
> Support for ligatures slows down character width calculations, which is why it hasn't been implemented. I'll reconsider this feature in version `3.x`, but cannot guarantee its inclusion.
> 
> Personally, I don't see much value in it, but I'm just a developer. Whether a feature is useful is up to the users. I remain neutral. Generally, I try to provide features users request.

I guess keeping it optional (and disabled in options by default) is going to be good enough?
